app/templates/main/error.hbs

```diff
@@ -1,7 +0,0 @@
-{{#if slug}}
```

Comment:
@drogus I just pushed up a new set of commits which I think should address this. The UI is slightly different, but I personally think it's an improvement. Instead of showing the loading indicator in the top left corner next to the sidebar, we use the same loading content as when the application boots. This makes transitioning between the routes feel seamless, and simply shows the error message in place of the loading indicator.

You can see this live in action on https://ce-ember-2-13.test-deployments.travis-ci.org/foobarbaz/barboofar.